src/parsers/manifest/metaplaylist/metaplaylist_parser.ts

```diff
@@ -293,13 +293,20 @@ function createManifest(
   }
 
   const time = performance.now();
+  let isLastPeriodKnown;
+  if (manifests.length > 0) {
+    isLastPeriodKnown = !isDynamic ||
+                        mplData.pollInterval === undefined &&
+                        manifests[manifests.length - 1].isLastPeriodKnown;
+  }
```

Comment:
I would say that `isLastPeriodKnown` would be set to `true` when there's no Period and `pollInterval` not set